examples/30_extended/tasks_tutorial.py

```diff
@@ -106,8 +103,7 @@
 # Finally, it is also possible to list all tasks on OpenML with:
 
 ############################################################################
-tasks = openml.tasks.list_tasks()
-tasks = pd.DataFrame.from_dict(tasks, orient='index')
+tasks = openml.tasks.list_tasks(output_format='dataframe')
 print(len(tasks))
```

Comment:
`logging.info()`?

Reply:
I disagree for the same reason as stated [here](https://github.com/openml/openml-python/pull/791/files#r330119323).